Cargo.toml

```diff
@@ -9,13 +9,16 @@ readme = "README.md"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
+cached = "0.20.0"
 cargo_metadata = "0.9"
 docopt = "1"
 env_logger = "0.8"
 html5ever = "0.24"
+lazy_static = "1.4.0"
```

Comment:
I would prefer to use `once_cell` over lazy_static - it uses regular functions and structs over macros and will also soon be part of the standard library: https://github.com/rust-lang/rfcs/pull/2788.